lib/integer.fz

```diff
@@ -60,6 +64,8 @@ integer(I (integer I).type) : numeric I is
       safety: other.sign >= 0
   is abstract
 
+  isBounded => false
```

Comment:
I think you should already use the "new" rules for underscores/case for identifiers. So `is_bounded` instead of `isBounded`. See here:
https://flang.dev/design/identifiers

Reply:
agreed. New names should be according to the rules. You might also rename existing names, but in a separate PR.